### PR TITLE
adhoc - Add references

### PR DESCRIPTION
**Pull request by Isengo1989**
Added security references

Noticed: Seems that we do not have a guide for custom recaptcha implementation (like in [SW5](https://developers.shopware.com/developers-guide/implementing-your-own-captcha/) )

### CODE REVIEW

**Comment by sushmangupta** [comment]
How about the inclusion of Captcha content that's here : https://shopware.atlassian.net/wiki/spaces/PLATFORM/pages/19487752365/Product+Security?focusedCommentId=19539231143?

**Comment by Isengo1989** [comment]
> How about the inclusion of Captcha content that's here : https://shopware.atlassian.net/wiki/spaces/PLATFORM/pages/19487752365/Product+Security?focusedCommentId=19539231143?

That part should have it's own article, IMO the info is too short.

File: resources/references/security.md [hunk 1]
```diff
@@ -4,10 +4,22 @@
...
+
+Shopware 6 provides certain rate limits by default that reduces the risk of brute-force attacks for pages like login or password reset. For more information, refer to [Rate Limiter](../../../guides/hosting/infrastructure/rate-limiter.md) guide.
+
 ## SameSite cookies
 
 SameSite prevents the browser from sending cookies along with cross-site requests. For more information on this refer to [SameSite Protection](../../../guides/hosting/configurations/framework/samesite-protection.md)
+
+## Security plugin
+
+Obtaining security fixes, without version upgrades is possible through the [Security plugin](../../../guides/hosting/installation-updates/cluster-setup#security-plugin)
```
Comment (sushmangupta) [inline resources/references/security.md:25]:
```suggestion
Obtaining security fixes, without version upgrades is possible through the [Security plugin](../../../guides/hosting/installation-updates/cluster-setup.md#security-plugin)
```

**Review by sushmangupta — APPROVED** [review]
@Isengo1989 , thanks for updating the information.

**Pull request merged**